Makefile

```diff
@@ -37,11 +37,9 @@ serve:
 
 install:
 	rm -f Gemfile.lock
-	test -f .bundle/config || mkdir -p .bundle; echo "---" > .bundle/config; echo 'BUNDLE_PATH: "vendor/gems"' >> .bundle/config
-	if ! bundle check; then bundle config set --local path 'vendor/gems'; bundle install; fi
+	test -f .bundle/config || mkdir -p .bundle; echo "---" > .bundle/config; echo 'BUNDLE_PATH: "vendor/gems"' >> .bundle/
+	if ! bundle check; then bundle config --local path 'vendor/gems'; bundle install; fi
```

Comment:
If the previous line is retained, the `bundle config --local path 'vendor/gems'` isn't necessary and could be removed; i.e.:
```
        if ! bundle check; then bundle install; fi
```

Reply:
Thanks for the corrections. I *think* this is finally ready to go.